### PR TITLE
chore(build): remove c++ runtime/stdlib dependency

### PR DESCRIPTION
**Pull request by eugenels**

### CODE REVIEW

**Review by bluestreak01 — DISMISSED** [review]

File: core/src/test/java/io/questdb/cairo/AbstractCairoTest.java [hunk 1]
```diff
@@ -155,7 +155,8 @@ public long getWriterAsyncCommandMaxTimeout() {
             public int getSqlJitMode() {
                 // JIT compiler is a beta feature and thus is disabled by default,
                 // but we want to have it enabled in tests.
-                return SqlJitMode.JIT_MODE_ENABLED;
+                //return SqlJitMode.JIT_MODE_ENABLED;
+                return SqlJitMode.JIT_MODE_DISABLED;
```
Comment (bluestreak01) [inline core/src/test/java/io/questdb/cairo/AbstractCairoTest.java:159]: Lets get to the bottom of the crash.

For context with JIT enabled this test crashes on JDK17

**Review by bluestreak01 — CHANGES_REQUESTED** [review]

File: core/src/main/c/share/cpprt_overrides.cpp [hunk 2]
```diff
@@ -0,0 +1,36 @@
...
+ ******************************************************************************/
+
+#include "cpprt_overrides.h"
+#include <stdlib.h>
+
+extern "C" void __cxa_pure_virtual() {
+    abort();
+}
+
+//void* operator new(size_t size) { return malloc(size); }
```
Comment (bluestreak01) [inline core/src/main/c/share/cpprt_overrides.cpp:32]: debris?

**Review by bluestreak01 — CHANGES_REQUESTED** [review]

**Review by bluestreak01 — DISMISSED** [review]

**Comment by ideoma** [comment]
#### [PR Coverage check]

:heart_eyes: **pass** : 1 / 1 (100.00%)




#### file detail

|   |path|covered line|new line|coverage|
|----|----|----|----|----|
|:large_blue_circle:|io/questdb/jit/JitUtil.java|1|1|100.00%|

**Comment by ideoma** [comment]
#### [PR Coverage check]

:heart_eyes: **pass** : 1 / 1 (100.00%)




#### file detail

|   |path|covered line|new line|coverage|
|----|----|----|----|----|
|:large_blue_circle:|io/questdb/jit/JitUtil.java|1|1|100.00%|

**Comment by ideoma** [comment]
#### [PR Coverage check]

:heart_eyes: **pass** : 1 / 1 (100.00%)




#### file detail

|   |path|covered line|new line|coverage|
|----|----|----|----|----|
|:large_blue_circle:|io/questdb/jit/JitUtil.java|1|1|100.00%|

**Comment by ideoma** [comment]
#### [PR Coverage check]

:heart_eyes: **pass** : 1 / 1 (100.00%)




#### file detail

|   |path|covered line|new line|coverage|
|----|----|----|----|----|
|:large_blue_circle:|io/questdb/jit/JitUtil.java|1|1|100.00%|

**Comment by ideoma** [comment]
#### [PR Coverage check]

:heart_eyes: **pass** : 1 / 1 (100.00%)




#### file detail

|   |path|covered line|new line|coverage|
|----|----|----|----|----|
|:large_blue_circle:|io/questdb/jit/JitUtil.java|1|1|100.00%|

**Comment by ideoma** [comment]
#### [PR Coverage check]

:disappointed: **fail** : 0 / 1 (00.00%)




#### file detail

|   |path|covered line|new line|coverage|
|----|----|----|----|----|
|:large_blue_circle:|io/questdb/jit/CompiledFilter.java|0|1|00.00%|

**Comment by ideoma** [comment]
#### [PR Coverage check]

:heart_eyes: **pass** : 1 / 1 (100.00%)




#### file detail

|   |path|covered line|new line|coverage|
|----|----|----|----|----|
|:large_blue_circle:|io/questdb/jit/CompiledFilter.java|1|1|100.00%|

**Comment by ideoma** [comment]
#### [PR Coverage check]

:disappointed: **fail** : 0 / 1 (00.00%)




#### file detail

|   |path|covered line|new line|coverage|
|----|----|----|----|----|
|:large_blue_circle:|io/questdb/jit/CompiledFilter.java|0|1|00.00%|

**Comment by ideoma** [comment]
#### [PR Coverage check]

:heart_eyes: **pass** : 1 / 1 (100.00%)




#### file detail

|   |path|covered line|new line|coverage|
|----|----|----|----|----|
|:large_blue_circle:|io/questdb/jit/CompiledFilter.java|1|1|100.00%|

**Comment by ideoma** [comment]
#### [PR Coverage check]

:heart_eyes: **pass** : 1 / 1 (100.00%)




#### file detail

|   |path|covered line|new line|coverage|
|----|----|----|----|----|
|:large_blue_circle:|io/questdb/jit/CompiledFilter.java|1|1|100.00%|

**Comment by ideoma** [comment]
#### [PR Coverage check]

:heart_eyes: **pass** : 1 / 1 (100.00%)




#### file detail

|   |path|covered line|new line|coverage|
|----|----|----|----|----|
|:large_blue_circle:|io/questdb/jit/CompiledFilter.java|1|1|100.00%|

**Comment by ideoma** [comment]
#### [PR Coverage check]

:disappointed: **fail** : 0 / 1 (00.00%)




#### file detail

|   |path|covered line|new line|coverage|
|----|----|----|----|----|
|:large_blue_circle:|io/questdb/jit/CompiledFilter.java|0|1|00.00%|

**Comment by ideoma** [comment]
#### [PR Coverage check]

:heart_eyes: **pass** : 1 / 1 (100.00%)




#### file detail

|   |path|covered line|new line|coverage|
|----|----|----|----|----|
|:large_blue_circle:|io/questdb/jit/CompiledFilter.java|1|1|100.00%|

**Review by bluestreak01 — APPROVED** [review]

**Pull request merged**